### PR TITLE
added footer test with anchor tags [Finishes #159499019 #159521207]

### PR DESCRIPTION
**Pull request by dnuwa**
hey guys, kindly spare some time to review my work 
@justinembawomye @nuwagalb @BarnaTB

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,21 +30,21 @@
 
                     </div>
                     <div>
-                        <input class="inner-input" type="" name="" placeholder="Phone number">
+                        <input input class="other-inputs" type="" name="" placeholder="Phone number">
```
Comment (nuwagalb) [inline index.html:33]: "input" word appears twice. Change it so that it appears once

File: index.html [hunk 2]
```diff
@@ -30,21 +30,21 @@
...
                     </div>
                     <div class="label">
                         <label>Enter your email (required)</label>
 
                     </div>
                     <div>
-                        <input type="" name="" placeholder="Email">
+                        <input input class="other-inputs" type="" name="" placeholder="Email">
```
Comment (nuwagalb) [inline index.html:40]: "input" word appears twice. Change it so that it appears once

File: index.html [hunk 3]
```diff
@@ -30,21 +30,21 @@
...
                     </div>
                     <div class="label">
                         <label>Enter a password (required)</label>
 
                     </div>
                     <div>
-                        <input type="" name="" placeholder="Password">
+                        <input input class="other-inputs" type="" name="" placeholder="Password">
```
Comment (nuwagalb) [inline index.html:47]: "input" word appears twice. Change it so that it appears once

**Review by nuwagalb — COMMENTED** [review]
Caught a few typo's in your code but otherwise it looks good

**Review by BarnaTB — COMMENTED** [review]
input appears twice so you might want to remove the second one, great work nonetheless

File: index.html (hunk 3 above)
Comment (BarnaTB) [reply index.html:47]: Also, you need to give that field a type of password to add some minimal HTML5 validation as opposed to leaving it blank

**Pull request merged**